mempool/src/pool/store.rs

```diff
@@ -64,6 +69,9 @@ pub struct MempoolStore {
     // rather a value that takes into account witdess and sigop data (see CTxMemPoolEntry::GetTxSize).
     pub(super) txs_by_descendant_score: BTreeMap<DescendantScore, BTreeSet<Id<Transaction>>>,
 
+    // Mempool entries sorted by ancestor score
```

Comment:
Maybe it's a good place to explain what ancestor score means

Reply:
To rephrase, who cares why transactions are removed?

Reply:
Updated the documentation.